other/DHT_bootstrap.c

```diff
@@ -193,7 +193,16 @@ int main(int argc, char *argv[])
 
     if (argc > argvoffset + 3) {
         printf("Trying to bootstrap into the network...\n");
-        uint16_t port = net_htons(atoi(argv[argvoffset + 2]));
+
+        const long int port_conv = strtol(argv[argvoffset + 2], nullptr, 10);
+
+        if (port_conv <= 0 || port_conv > (uint16_t) -1) {
+            printf("Failed to convert \"%s\" into a valid port. Exiting...\n", argv[argvoffset + 2]);
```

Comment:
strkey does fprintf stderr. Should we do that here too?

Reply:
I just followed the rest of the file which uses `printf`